types/artifacts.d.ts

```diff
@@ -225,6 +225,8 @@ declare global {
       export interface ScriptElement {
         type: string | null
         src: string | null
+        /** The id uniquely identifies a script element, null if script source is network */
```

Comment:
one possible rewording to also include the case of if the element had no `id`
```suggestion
        /** The `id` property of the script element; null if it had no `id` or if `source` is 'network'. */
```